lib/FusionInventory/Agent/Tools/SNMP.pm

```diff
@@ -113,10 +113,10 @@ sub getCanonicalMemory {
     # Don't try to analyse negative values
     return if $value =~ /^-/;
 
-    if ($value =~ /^(\d+) KBytes$/) {
-        return int($1 / 1024);
+    if ($value =~ /^(\d+) (KBytes|kB)$/) {
+        return int($1 / 1_048.576);
```

Comment:
Why changing the divisor to `1_048.576` here ?

Reply:
In [network inventory protocol](http://fusioninventory.org/documentation/dev/spec/protocol/netinventory.html), value wanted is Mio.
Mio is binary unit : 2^20.
kB is decimal unit : 10^3.

So to get Mio from kB we need divide by 2^20/10^3=1048.576 .

Reply:
Well if I read this [KBytes definition on host mib](http://www.net-snmp.org/docs/mibs/host.html#KBytes), I would better expect a by 1024 division as it was before.
Tell me where I'm wrong.

Reply:
Right 1024 is the right division